tests/integration/cassettes/TestSubmission.test_hide.json

```diff
@@ -1,7 +1,7 @@
 {
   "http_interactions": [
     {
-      "recorded_at": "2016-07-12T18:08:50",
```

Comment:
Can you remove the changes to this file from your PR? It looks like nothing of importance should have changed as a result of your updates.